cloudbuild.yaml

```diff
@@ -0,0 +1,72 @@
+# This cloudbuild.yaml config file is triggered in two contexts:
+#
+#
+# Trigger #1: Before master merge
+#
+# marketplace-k8s-app-tools push to any branch
+# https://console.cloud.google.com/cloud-build/triggers/0762301f-81af-4c60-8d12-a4cbdf8d6ea1?project=marketplace-k8s-app-tools
+#
+# This build trigger asserts the invariant that the source code can
+# be built and pushed into a repository. Its success blocks pull
+# request. gcr.io/marketplace-k8s-app-tools read access restricted.
+#
+#
+# Trigger #2: After master merge
+#
+# cloud-marketplace-tools push to master branch
+# https://console.cloud.google.com/cloud-build/triggers/0257675b-9860-40a0-90c3-640a196e6a56?project=cloud-marketplace-tools
+#
+# This build trigger asserts the invariant that each commit on the master
+# branch of GoogleCloudPlatform/marketplace-k8s-app-tools will be built
+# and tagged in gcr.io/cloud-marketplace-tools/master.
+steps:
```

Comment:
Add a note here to document which env variables are built-ins (seems like all of them are).

Reply:
Unless this is not the auto-push/auto-release workflow? I understand this to be that workflow though.

Reply:
Done.